docs/tutorial/tutorial-5/android.rst

```diff
@@ -2,9 +2,293 @@
+
+      >
+
+Choose your desired device, or choose the option to create a new device. If you create a new
+device, you will have to provide a name (or accept the default of ``beePhone``).
+
+Briefcase ``run`` will automatically boot the virtual device if necessary. When the device
+is booting, you will see the Android logo.
+
+.. figure:: ../images/android/tutorial-5-booting.png
+   :alt: Android virtual device booting
+
+   Android virtual device booting
+
+Briefcase ``run`` will start your app when the device finishes booting. You will briefly
+see a launcher screen.
+
+.. figure:: ../images/android/tutorial-5-running.png
+   :alt: Android virtual device fully started, on the launcher screen
+
+   Android virtual device fully started, on the launcher screen
+
+Briefcase ``run`` will proceed to launch your app with no intervention required by you.
+BeeWare Android apps begin with a splash screen while the Python components
+initialize.
+
+.. figure:: ../images/android/tutorial-5-splash.png
+   :alt: App splash screen
+
+   App splash screen
+
+The app will initialize Python and draw your widgets. In the case of the
+app built in :doc:`Tutorial 2 <../tutorial-2>`, it will look like this.
+
+.. figure:: ../images/android/tutorial-5-launched.png
+   :alt: App from Tutorial 2, fully launched
+
+   App from Tutorial 2, fully launched
+
+If you fail to see your app launching, you may need to check your terminal
+where you ran ``briefcase run`` and look for any error messages.
+
+Run the app on a physical device
+================================
+
+If you have an Android phone or tablet you want to run your app on, you can
+connect it to your development workstation, typically using a USB cable.
+Then you can use Briefcase's ``run`` command to target your physical device.
+
+If Briefcase can detect the device, it will appear in the ``run`` output. The
+following output shows what you might see if you have a Google Pixel 3a device
+connected.
+
+Although Android devices can be given names, the name that appears here
+is typically the
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase run android
+
+      Select device:
+
+        1) Pixel_3a (94ZZY0LNE8)
+        2) Create a new Android emulator
+
+      >
+
+  .. group-tab:: Linux
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase run android
+
+      Select device:
+
+        1) Pixel_3a (94ZZY0LNE8)
+        2) Create a new Android emulator
+
+      >
+
+  .. group-tab:: Windows
+
+    .. code-block:: doscon
+
+      Select device:
+
+        1) CoolPhone
+        2) Create a new Android emulator
+
+      >
+
+In this example, you could type **1** then press return to run the app on your
+Pixel 3a device. When creating this list, Briefcase queries `adb
+<https://developer.android.com/studio/command-line/adb>`__ for the device's
+serial number (in this case, 94ZZY0LNE8) and model name (in this case,
+Pixel_3a).
```

Comment:
See the cosmetic note about naming on beeware/briefcase#433 - if you decide to make that change, there's some notes here that need to be updated.